lib/src/cpp/utils/ParseXMLErrorHandler.cpp

```diff
@@ -46,25 +46,28 @@ void ParseXMLErrorHandler::fatalError(const xercesc::SAXParseException& ex)
     report_parse_exception(ex);
 }
 
-void ParseXMLErrorHandler::resetErrors() { }
+void ParseXMLErrorHandler::resetErrors()
+{
+
+}
```

Comment:
Why do we need this method if it has an empty implementation?

Reply:
Override...